packages/testcontainers/src/MetaChainContainer.ts

```diff
@@ -37,15 +47,13 @@ export class MetaChainContainer extends GenericContainer {
   }
 
   public async start(): Promise<StartedMetaChainContainer> {
-    const network = await new Network().start();
-
-    this.withExposedPorts(...Object.values(this.config.ports))
-      .withNetworkMode(network.getName())
-      .withCmd(this.getCmd())
-      .withStartupTimeout(120_000);
```

Comment:
```ts
    this.withExposedPorts(...Object.values(this.config.ports))
      .withCmd(this.getCmd());
```

Keep these. Otherwise, there is a race condition. Since `withNetworkConfig()` should override the `getCmd` and exposed ports which aren't evaluated if it's applied only at the constructor.